src/routes/tweets.route.ts

```diff
@@ -0,0 +1,43 @@
+// eslint-disable-next-line import/order
+import express, { Request, Response } from "express";
+
+import * as tweetsService from "../services/tweets.sevice";
+
+export const tweetsRouter = express.Router();
+tweetsRouter.get("/getOne/:id", async (req: Request, res: Response) => {
+  const tweet = await tweetsService.getTweetById(req.params.id);
+  res.json(tweet);
+});
+
+tweetsRouter.get("/getAll", async (req: Request, res: Response) => {
```

Comment:
I think good practice is to make routes like so: 
- when you get all or post: .get("/") or ,post("/")
- when you get one  by id or delete: ,get("/:id")  or .delete("/:id")